__tests__/schema/convert-edtr-io-state.ts

```diff
@@ -67,6 +67,7 @@ describe('edtr io plugins', () => {
     })
   })
 
+  //TODO: Not supported in editor any more, should we do something here? (like make it bold at least?)
```

Comment:
Actually it is supported, see https://github.com/serlo/serlo.org/blob/6eca3d8c02994f5d0356d8bd73907a9afb3fd293/packages/private/edtr-io/src/plugins.tsx#L109 and https://github.com/serlo/serlo.org/blob/6eca3d8c02994f5d0356d8bd73907a9afb3fd293/packages/private/edtr-io/src/editor.tsx#L212.

Reply:
@inyono Thanks! Issue in #597